src/Auth0.ManagementApi/Models/UserCreateRequest.cs

```diff
@@ -20,7 +20,7 @@ public class UserCreateRequest : UserBase
         public string UserId { get; set; }
 
         /// <summary>
-        /// Gets or sets the user's password. This is mandatory on non-SMS connections.
+        /// Gets or sets the user's password. This is mandatory on non SMS or email connections.
```

Comment:
Soon the docs will contain the following:

> Initial password for this user (mandatory only for auth0 connection strategy).

So lets update this to be:

```suggestion
/// Gets or sets the user's password. This is mandatory only for auth0 connection strategy.
```